_posts/2019-08-20-service-objects.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "Service Objects: beyond fat models and skinny controllers"
+date: 2019-09-03 10:00:00
+categories: ["rails"]
```

Comment:
@cleicar you can add `code-refactor` too. you have a few articles in that category already